ci-operator/step-registry/osd-ccs/cluster/provision/gcp/osd-ccs-cluster-provision-gcp-commands.sh

```diff
@@ -93,9 +173,12 @@ echo "  Etcd encryption: ${ETCD_ENCRYPTION}"
 echo "  Disable workload monitoring: ${DISABLE_WORKLOAD_MONITORING}"
 echo "  Subscription type: ${SUBSCRIPTION_TYPE}"
 echo "  Secure boot for shielded VMs: ${SECURE_BOOT_FOR_SHIELDED_VMS}"
+echo "  VPC project id: ${VPC_PROJECT_ID:-}"
```

Comment:
I think the condition `if [ "${ENABLE_SHARED_VPC}" == "yes" ]; then` is perfect. My previous comment is advising remove `echo "$(date -u --rfc-3339=seconds) - INFO: Install into GCP Shared VPC" ` but not this `if`.